generative_data_prep/utils/logger_config.ini

```diff
@@ -0,0 +1,30 @@
+[loggers]
+keys=root,generative_data_prep_logger
+
+[handlers]
+keys=consoleHandler
+
+[formatters]
+keys=fileFormatter,consoleFormatter
+
+[logger_root]
+level=DEBUG
+handlers=consoleHandler
+propagate=0
+
+[logger_generative_data_prep_logger]
+level=DEBUG
+handlers=consoleHandler
+qualname=generative_data_prep_logger
+propagate=0
+
+[handler_consoleHandler]
+class=StreamHandler
+level=INFO
+formatter=consoleFormatter
+
+[formatter_fileFormatter]
+format=%(message)s
```

Comment:
are we sure this is what we want? Generally we want a little more information:

```
format=%(asctime)s %(levelname)s:%(name)s:%(message)s

Reply:
Yes, this is not a debug logger, it is to easily get information about what kind of dataset was prepared.

Reply:
Well, standards exist for a reason. If we don't want all that info by default, then we can create additional handlers. An example config might look like this:

```
[handlers]
keys=consoleHandler,infoFileHandler,debugFileHandler

[formatters]
keys=fileFormatter,consoleFormatter,infoFormatter,debugFormatter

[handler_infoFileHandler]
class=FileHandler
level=INFO
formatter=infoFormatter
args=('info.log', 'a')

[handler_debugFileHandler]
class=FileHandler
level=DEBUG
formatter=debugFormatter
args=('debug.log', 'a')

[formatter_infoFormatter]
format=%(asctime)s - %(name)s - %(levelname)s - %(message)s
datefmt=%Y-%m-%d %H:%M:%S

[formatter_debugFormatter]
format=%(asctime)s - %(name)s - %(levelname)s - [%(filename)s:%(lineno)d] - %(message)s
datefmt=%Y-%m-%d %H:%M:%S

Reply:
Yeah makes sense, we can always add more handlers in the future with more information!